### ISSUE TITLE
Input file processing

### ISSUE THREAD

**Issue by ModiShrushti** [swidler/LINADMIX#1]
Hi,
I would like to know about how to process the input file for the target and source population. Does it require running admixture and plink on all the ancient and modern samples together?

**Comment by random139847585** [comment]
Yes, samples should be in 1 dataset - input files are a .raw file of the dataset (using PLINK), .fam file for the dataset, .ind file for the dataset, and the output files of an ADMIXTURE run for the dataset.
